packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1332,15 +1332,13 @@ Adds an OAuth 2.0 client application. This application is only available to the
 
 * At least one redirect URI and response type is required for all client types, with exceptions: if the client uses the [Resource Owner Password](https://tools.ietf.org/html/rfc6749#section-4.3) flow (if `grant_types` contains the value `password`) or [Client Credentials](https://tools.ietf.org/html/rfc6749#section-4.4) flow (if `grant_types` contains the value `client_credentials`) then no redirect URI or response type is necessary. In these cases you can pass either null or an empty array for these attributes.
 
-* If `wildcard_redirect` is `DISABLED`, all redirect URIs must be absolute URIs and must not include a fragment component.
+* If `wildcard_redirect` <ApiLifecycle access="ea" /> is `DISABLED`, all redirect URIs must be absolute URIs and must not include a fragment component. If `wildcard_redirect` is `SUBDOMAIN`, then configured redirect URIs may contain a single `*` character in the lowest-level domain to act as a wildcard. The wildcard subdomain must have at least one subdomain between it and the top level domain.
```

Comment:
Do you want to use `mustn't` here? I'm always leery on contractions, so you're the arbiter.

Reply:
Gah - I'd rather not - besides it MUST have not must NOT